doc/content/source/auxkernels/DensityNormalization.md

```diff
@@ -10,7 +10,13 @@ documentation clear for users.
 
 ## Overview
```

Comment:
@gsgall The reason the the undocumented class warning was still there was because this auxkernel, and several other objects, don't have a formalized test. We could remove the overall undocumented class warning, but if we do that, I would like to add an alert warning that states there is no active test for the auxkernel.

@cticenhour is there a standard approach with MOOSE documentation for objects without test examples?

Reply:
The standard approach is that all objects should be tested. 😉 

But in all seriousness, I think the undocumented class warning should be removed if Overview and Example Input Syntax is filled in (so I would fill in the latter section, since there isn't a test input otherwise providing one in the `!syntax inputs` section of the file).

In place of the undocumented class warning, I agree with @csdechant that a warning alert that the object isn't tested should be added. It's important for users to know this. We don't necessarily do this for MOOSE, but that's because we require tests and documentation for everything added these days. Zapdos is playing catch-up, so I think that's an appropriate rule until such time as we can get everything tested fully.

Reply:
In the body of these warnings I think we should also have some basic directions on how users can report bugs found in undocumented classes or contribute tests for these classes to Zapdos as well.

Reply:
Warnings have been added. Within the warnings, there are links to the Zapdos Discussions page and the MOOSE Contributing page for starting points on how users can discuss problems and contribute to Zapdos/MOOSE.